patch_api/api/estimates_api.py

```diff
@@ -41,8 +41,11 @@ class EstimatesApi(object):
         "make",
         "year",
         "transaction_value_btc_sats",
-        "timestamp",
+        "transaction_value_eth_gwei",
```

Comment:
Looks like we (I) had forgotten a couple parameters. This is why I'm trying to refactor the python SDK generation :)